hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/BPServiceActor.java

```diff
@@ -202,6 +202,7 @@ private String getNameNodeAddress() {
   Map<String, String> getActorInfoMap() {
     final Map<String, String> info = new HashMap<String, String>();
     info.put("NamenodeAddress", getNameNodeAddress());
+    info.put("NamenodeState", state.toString());
```

Comment:
nit: is it better to name it `NameNodeHaState`

Reply:
Actually that is better, since that is the main purpose we want to debug. Let me make this change.